exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -285,21 +305,28 @@ fn parse_service_specs(
 fn parse_instance_spec(
     env: &JNIEnv,
     service_spec_obj: JObject,
-    artifact_id: impl AsRef<str>,
+    artifact_id: ArtifactId,
 ) -> JniResult<(InstanceSpec, Vec<u8>)> {
     let (service_id, service_name) = get_service_id_and_name(env, service_spec_obj)?;
-    let config_params: jbyteArray = env
-        .get_field(service_spec_obj, "configuration", "[B")?
-        .l()?
-        .into_inner();
-    let config = env.convert_byte_array(config_params)?;
-    let spec = InstanceSpec::new(service_id, service_name, artifact_id).map_err(|err| {
+
+    let spec = InstanceSpec {
```

Comment:
InstanceSpec::new is absolutely useless as it accepts `String` for `artifact_id`